src/ctelnet.cpp

```diff
@@ -2028,3 +2156,57 @@ void cTelnet::setKeepAlive(int socketHandle)
     setsockopt(socketHandle, IPPROTO_TCP, TCP_KEEPCNT, &count, sizeof(count));
 #endif // defined(Q_OS_WIN32)
 }
+
+// Used to convert a collection of Bytes in the current MUD Server encoding
+// to the UTF-16BE encoding used for QString and then back to a series of bytes
+// as a QByteArray - note that it does NOT retain states between calls as it
+// assumes each call is a complete separate chunk of text - should this not
+// prove to be the case in practice it will be necessary to fork-off seperate
+// instances of this method for each OOB protocol that uses this DECODER:
+QByteArray cTelnet::decodeBytes(const char* bytes)
+{
+    // Check this each packet
+    QString usedEncoding = mpHost->mTelnet.getEncoding();
```

Comment:
Is a getter function really necessary here or just unnecessary overhead?

Reply:
Ah, it is an unintentional bit of copy-pasting of code from elsewhere and can be cut-out here - it is, in fact, pointless...!

Reply:
Fixed